Reaktoro/Core/Phases.py.cxx

```diff
@@ -45,6 +53,8 @@ void exportPhases(py::module& m)
         .def(py::init<>())
         .def(py::init<const StringList&>())
         .def(py::init<const Speciate&>())
+        .def(py::init<const Speciate&, const Exclude&>())
```

Comment:
We need:

~~~cpp
.def(py::init<const Speciate&, const Exclude&>())
.def(py::init<const Exclude&>())
~~~

for `GenericPhaseGenerator` too.